### ISSUE TITLE
Short reporter block names don't center correctly

### ISSUE THREAD

**Issue by husqwc** [scratchblocks/scratchblocks#506]
List and normal block names don't center in Scratch
<img width="90" alt="Screenshot 2024-02-19 at 2 37 19 PM" src="https://github.com/scratchblocks/scratchblocks/assets/105561911/f0f49dbe-db9f-41bc-aafe-08d5b960cf9e">
but they always centers with scratchblocks
![scratchblocks](https://github.com/scratchblocks/scratchblocks/assets/105561911/db18a625-cdaf-4876-8ed4-562b36c7b6b5)

**Comment by tjvr** [comment]
Interesting! I’m sure this was right when I first built it, so Scratch itself must have changed.

**Comment by ego-lay-atman-bay** [comment]
Why this inconsistency?

**Comment by husqwc** [comment]
I'm sure no one intentionally wrote lines of code specifically so that variable names center while lists don't, but even if it were to be a bug, it's been like this since the 3.0 beta preview (in fact I can't find a single 3.0 version where list names are centered), so it should probably be fixed.

**Comment by ego-lay-atman-bay** [comment]
Well in that case, I guess it might be by (inconsistent) design.

**Comment by tjvr** [comment]
@husqwc Can you give an example of a reporter block (i.e. not a variable, list or parameter) with a very short name, that would be affected by this?

**Comment by tjvr** [comment]
Aside: bafflingly, it looks like short boolean parameters are _right_-aligned??

I am not going to fix that :see_no_evil:  

![screenshot_2024-04-14-172500](https://github.com/scratchblocks/scratchblocks/assets/639289/f311e593-ded5-43c1-8d53-336c5ae832b5)

**Comment by husqwc** [comment]
> @husqwc Can you give an example of a reporter block (i.e. not a variable, list or parameter) with a very short name, that would be affected by this?

I've mentioned the reporter thing here https://github.com/scratchblocks/scratchblocks/pull/510#discussion_r1565766927.

As for booleans, there is no boolean block that is short enough to trigger the minimum block limit (the closest thing I found is the <លឺ?> block). I don't know what it would look like for a normal boolean block to have a minimum length (you'd have to ask @CST1229 for that), but ideally the scratchblocks should mimic Scratch implementation (even when Scratch's implementation is not the best) for stuff like extension documentation.

**Comment by CST1229** [comment]
> As for booleans, there is no boolean block that is short enough to trigger the minimum block limit (the closest thing I found is the <លឺ?> block). I don't know what it would look like for a normal boolean block to have a minimum length (you'd have to ask @CST1229 for that), but ideally the scratchblocks should mimic Scratch implementation (even when Scratch's implementation is not the best) for stuff like extension documentation.

Text alignment is normal for normal blocks:
![image](https://github.com/scratchblocks/scratchblocks/assets/68464103/a8d192c9-399d-4ee3-b3a1-153a85fc31d7)


The minimum width of a boolean seems to be 12 grid units (or 48 1x pixels).
https://github.com/scratchfoundation/scratch-blocks/blob/ed02f0c01bd43bcf403988182a268262a1586d9c/core/block_render_svg_vertical.js#L63
![image](https://github.com/scratchblocks/scratchblocks/assets/68464103/3a1bceeb-5534-4644-a561-185fc489bbd8)

**Comment by tjvr** [comment]
> ideally the scratchblocks should mimic Scratch implementation (even when Scratch's implementation is not the best)

You're right in principle — matching Scratch's rendering, to within about a pixel, is absolutely a goal of scratchblocks.

However, the seemingly right-aligned boolean parameters seems like it _has_ to be a bug in Scratch?? I'm not sure I even understand what I'm looking at — the text appears to be located where I'd expect the right padding to be!

**Comment by husqwc** [comment]
After the April 2026 editor update, the centering have changed so short list blocks and normal blocks are centered. However, normal reporters are now not centered properly (seems like a bug? hopefully they fix it):
<img width="86" height="119" alt="Image" src="https://github.com/user-attachments/assets/59dc0a5f-48f0-4b30-afec-878e47783612" />
